statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -646,20 +828,35 @@ def bic(self):
     @cache_readonly
     def cov_params_default(self):
```

Comment:
I have this defined as simple attribute not a cached property in other models, and set it in the Results `__ini__`

I guess this works also.